tests/Fixer/Casing/NativeFunctionTypeDeclarationCasingFixerTest.php

```diff
@@ -184,9 +194,31 @@ public function testFix81(string $expected, string $input): void
 
     public function provideFix81Cases(): \Generator
     {
-        yield [
+        yield 'return type `never`' => [
             '<?php class T { public function Foo(object $A): never {die;}}',
             '<?php class T { public function Foo(object $A): NEVER {die;}}',
         ];
     }
+
+    /**
+     * @dataProvider provideFix82Cases
+     * @requires PHP 8.2
```

Comment:
We don't run PHP 8.2 on GitHub Actions yet, I think this is a good opportunity to add it.

Reply:
```
Specifying 8.2 in php-version input installs a nightly build of PHP 8.2.0-dev. See [nightly build setup](https://github.com/marketplace/actions/setup-php-action#nightly-build-setup) for more information.
```

https://github.com/marketplace/actions/setup-php-action#tada-php-support

do we want to add the `nightly` build (yet) and maybe keep it in future as an optional step?

Reply:
maybe we could, so we can see this test running?

Reply:
@julienfalque @keradus it's covered in #6558